CHANGES.md

```diff
@@ -1,112 +1,23 @@
-
-* Bumped Telegraf to have Mesos overlay module metrics collected. (DCOS_OSS-5323)
-
-* Add more vm metrics to dcos-net. (DCOS_OSS-5335)
-
-* Introduced a new DC/OS configuration parameter `mesos_docker_volume_chown`, by default as `false`. If this parameter is set as `true`, Mesos will change the ownership of a docker volumes non-recursively to be the task user when launching a container. Please notice that this parameter is NOT recommended to switch on if there is any docker volume shared by multiple non-root users. (DCOS_OSS-5381)
-
-* Introduced a new DC/OS configuration parameter `mesos_disallow_sharing_agent_ipc_namespace`, defaulting to `false`. This parameter can be used to control whether the top-level Mesos container is allowed to share Mesos agent host's IPC namespace and /dev/shm. (DCOS-56619)
-
-* Introduced a new DC/OS configuration parameter `mesos_default_container_shm_size`. This parameter can be used to specify the default size of the /dev/shm for the Mesos container which has its own /dev/shm. The format is `[number][unit]`, `number` must be a positive integer and `unit` can be B (bytes), KB (kilobytes), MB (megabytes), GB (gigabytes) or TB (terabytes). (DCOS-56619)
-
-* Add dcos-net overlay metrics. (DCOS_OSS-5324)
-
-* Added containerizer debug endpoint into the diagnostic bundle. This endpoint is used for tracking data for stuck tasks. (DCOS-55383)
-
-* Prune VIPs with no backends in order to avoid unbounded growth of state and messages exchanged among `dcos-net` processes. (DCOS_OSS-5356)
-
-* DC/OS no longer increases the rate limit for journald logging.  Scale testing demonstrated that raising the limit overloads journald, causing problems for other components that see delayed or lost logs or, worse, hang until log buffers are read. The default of 10000 messages per 30 seconds appears to distinguish well between busy components and excessively verbose components. (DCOS-53763)
-
-* DC/OS Net: Fix support for big sets in the ipset manager. (COPS-5229)
-
-* DC/OS Net: switch to Erlang/OTP's Logger in order to be able to handle log message bursts without compromising the system stability. (DCOS_OSS-5461)
-
-* DC/OS Net: use exponential backoff when retrying failed requests to Mesos in order not to impose additional load onto potentially already overloaded Mesos. (DCOS_OSS-5459)
-
 * Mesos overlay networking: support dropping agents from the state. (DCOS_OSS-5536)
 
 ### Breaking changes
 
-The following parameters have been removed from the DC/OS installer:
-
-* --set-superuser-password
-* --offline
-* --cli-telemetry-disabled
-* --validate-config
-* --preflight
-* --install-prereqs
-* --deploy
-* --postflight
-
-The Marathon option `MARATHON_DEFAULT_ACCEPTED_RESOURCE_ROLES` has been deprecated. Please use `MARATHON_ACCEPTED_RESOURCE_ROLES_DEFAULT_BEHAVIOR`, instead, which has valid values of `any`, `unreserved`, or `reserved`.
 
 ### Fixed and improved
```

Comment:
Hey Senthil, this section should not be here.

It was accidentally added by this PR:
https://github.com/dcos/dcos/commit/47bcb77eb91fc707e9b0b3c120875dde3a2db747#r35098787

Reply:
Thanks for the review, JP. I have addressed it here: https://github.com/dcos/dcos/pull/6264